src/assets/v1.3.0/sign_message_lib.json

```diff
@@ -30,6 +30,8 @@
     "246": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "288": "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
     "300": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
+    "321": "0x03F886722b44BefB13871D4a05621D38616D3b7c",
```

Comment:
I couldn't validate succesfully this contract, could you review? Maybe wrong branch on safe-contracts? 
https://github.com/safe-global/safe-contracts#deploy

> Current version: The latest release is [v1.3.0-libs.0](https://github.com/gnosis/safe-contracts/tree/v1.3.0-libs.0) on the commit [767ef36](https://github.com/gnosis/safe-contracts/commit/767ef36bba88bdbc0c9fe3708a4290cabef4c376)

Reply:
I also failed to verify, it may be a explorer bug

<img width="975" alt="image" src="https://user-images.githubusercontent.com/32643286/195068442-2c8ffe45-5741-4ddc-a0b7-56e495896547.png">